tests/user_commands/tests.py

```diff
@@ -247,6 +247,13 @@ def test_mutually_exclusive_group_required_options(self):
         with self.assertRaisesMessage(CommandError, msg):
             management.call_command('mutually_exclusive_required', stdout=out)
 
+    def test_mutually_exclusive_group_required_with_boolean(self):
+        out = StringIO()
+        management.call_command('mutually_exclusive_required_with_boolean', '--bar', stdout=out)
+        self.assertIn('bar', out.getvalue())
+        management.call_command('mutually_exclusive_required_with_boolean', bar=True, stdout=out)
+        self.assertIn('bar', out.getvalue())
```

Comment:
`'bar'` is already in `out` from the first execution of `call_command()`. You should reinstantiate or use `out.truncate(0)` before the second call.